orga/app/components/password-reset-modal.js

```diff
@@ -2,6 +2,7 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 import Component from '@glimmer/component';
+import _ from 'lodash';
```

Comment:
Attention dans Pix Orga, il faut importer seulement les méthodes qu'on utilise de Lodash.
```suggestion
import get from 'lodash/get';
```

Reply:
Done